[7.13.x-blue] [KIECLOUD-647] Remove OpenShift templates from rhpam-7-openshift-image

davidesalerno opened the pull request:
Signed-off-by: Davide Salerno <dsalerno@redhat.com>

See: [KIECLOUD-647](https://issues.redhat.com/browse/KIECLOUD-647)

Thanks for submitting your Pull Request!

Please make sure your PR meets the following requirements:

- [x] Pull Request title is properly formatted: `[RHPAM-XYZ] Subject`
- [x] Pull Request contains link to the JIRA issue
- [x] Pull Request contains description of the issue
- [x] Pull Request does not include fixes for issues other than the main ticket
- [x] Attached commits represent units of work and are properly formatted
- [x] You have read and agreed to the Developer Certificate of Origin (DCO) (see `CONTRIBUTING.md`)
- [x] Every commit contains `Signed-off-by: Your Name <yourname@example.com>` - use `git commit -s`

spolti commented on README.md:
```diff
@@ -32,10 +28,9 @@ Inside each image directory you will find the following files:
 
 
-On the root directory you can also find the two files:
+On the root directory you can also one additiona file:
```
```suggestion
On the root directory you can also find one additional file:
```

spolti commented on quickstarts/hello-rules-multi-module/README.md:
```diff
@@ -16,69 +17,30 @@ $ oc login https://<your_openshift_address>:<port>
 
-At this moment we are ready to instantiate the kieserver app:
+To create the IBM BAMOE environment on your OpenShift instance you need to install Business Automation Operator.
+For instructions about installing Business Automation Operator, see [the product documentation](https://access.redhat.com/documentation/en-us/red_hat_process_automation_manager/7.8/html/deploying_a_red_hat_process_automation_manager_environment_on_red_hat_openshift_container_platform_using_operators/operator-con#operator-subscribe-proc) step.
```
```suggestion
For instructions about installing Business Automation Operator, see [the product documentation](https://access.redhat.com/documentation/en-us/red_hat_process_automation_manager/7.12/html/deploying_a_red_hat_process_automation_manager_environment_on_red_hat_openshift_container_platform_using_operators/operator-con#operator-subscribe-proc) step.
```

davidesalerno replied:
+1

spolti commented on quickstarts/hello-rules-multi-module/README.md:
```diff
@@ -16,69 +17,30 @@ $ oc login https://<your_openshift_address>:<port>
+To create the IBM BAMOE environment on your OpenShift instance you need to install Business Automation Operator.
+For instructions about installing Business Automation Operator, see [the product documentation](https://access.redhat.com/documentation/en-us/red_hat_process_automation_manager/7.8/html/deploying_a_red_hat_process_automation_manager_environment_on_red_hat_openshift_container_platform_using_operators/operator-con#operator-subscribe-proc) step.
 
+Then access the wizard installer, instructions available [here](https://access.redhat.com/documentation/en-us/red_hat_process_automation_manager/7.8/html/deploying_a_red_hat_process_automation_manager_environment_on_red_hat_openshift_container_platform_using_operators/operator-con#operator-deploy-start-proc)
```
```suggestion
Then access the wizard installer, instructions available [here](https://access.redhat.com/documentation/en-us/red_hat_process_automation_manager/7.12/html/deploying_a_red_hat_process_automation_manager_environment_on_red_hat_openshift_container_platform_using_operators/operator-con#operator-deploy-start-proc)
```

spolti commented on quickstarts/hello-rules-multi-module/README.md:
```diff
@@ -16,69 +17,30 @@ $ oc login https://<your_openshift_address>:<port>
--p SOURCE_REPOSITORY_REF=main \
--p CONTEXT_DIR=quickstarts/hello-rules-multi-module \
--p IMAGE_STREAM_NAMESPACE=openshift
+If you view the generated YAML source for the installation, the source should be similar to the following example:
```
```suggestion
If you view the generated YAML source for the installation, the source should be similar to the following example:

```

spolti commented on quickstarts/hello-rules-multi-module/README.md:
```diff
@@ -16,69 +17,30 @@ $ oc login https://<your_openshift_address>:<port>
 
-
-Now you can deploy the [hellorules-client](hellorules-client) in the same or another project and test IBM BAMOE KIE Server container.
+When your KieApp will be ready and running, you can go ahead deploying  the [hellorules-client](hellorules-client) in the same or another project and test IBM BAMOE KIE Server container.
```
```suggestion
When the KieApp gets ready, you can go ahead deploying  the [hellorules-client](hellorules-client) in the same or another project and test IBM BAMOE KIE Server container.
```

spolti commented on quickstarts/hello-rules/README.md:
```diff
@@ -1,14 +1,14 @@
 ## IBM Business Automation Manager Open Editions KIE Server decisions Quickstart
 
-Note that, this is the development branch, the target images might not be available here, instead you can look at the [released branch](https://github.com/jboss-container-images/rhpam-7-openshift-image/tree/7.13.x-blue/quickstarts/hello-rules)
+Note that, this is the development branch, the target images might not be available here, instead you can look at the [released branch](https://github.com/jboss-container-images/rhpam-7-openshift-image/tree/7.13.x/quickstarts/hello-rules)
```
```suggestion
Note that, this is the development branch, the target images might not be available here, instead you can look at the [released branch](https://github.com/jboss-container-images/rhpam-7-openshift-image/tree/7.13.x-blue/quickstarts/hello-rules)
```

spolti commented on quickstarts/hello-rules/README.md:
```diff
@@ -20,67 +20,30 @@ $ oc login https://<your_openshift_address>:<port>
 please take a look [here](https://access.redhat.com/RegistryAuthentication)
 
+To create the IBM BAMOE environment  on your OpenShift instance you need to install Business Automation Operator.
+For instructions about installing Business Automation Operator, see [the product documentation](https://access.redhat.com/documentation/en-us/red_hat_process_automation_manager/7.8/html/deploying_a_red_hat_process_automation_manager_environment_on_red_hat_openshift_container_platform_using_operators/operator-con#operator-subscribe-proc) step.
```
```suggestion
For instructions about installing Business Automation Operator, see [the product documentation](https://access.redhat.com/documentation/en-us/red_hat_process_automation_manager/7.13/html/deploying_a_red_hat_process_automation_manager_environment_on_red_hat_openshift_container_platform_using_operators/operator-con#operator-subscribe-proc) step.
```

spolti commented on quickstarts/hello-rules/README.md:
```diff
@@ -20,67 +20,30 @@ $ oc login https://<your_openshift_address>:<port>
-Default credential is `adminUser/RedHat`.
-
-At this moment we are ready to instantiate the kieserver app:
+Then access the wizard installer, instructions available [here](https://access.redhat.com/documentation/en-us/red_hat_process_automation_manager/7.8/html/deploying_a_red_hat_process_automation_manager_environment_on_red_hat_openshift_container_platform_using_operators/operator-con#operator-deploy-start-proc)
```
```suggestion
Then access the wizard installer, instructions available [here](https://access.redhat.com/documentation/en-us/red_hat_process_automation_manager/7.13/html/deploying_a_red_hat_process_automation_manager_environment_on_red_hat_openshift_container_platform_using_operators/operator-con#operator-deploy-start-proc)
```

spolti commented on quickstarts/hello-rules/README.md:
````diff
@@ -125,35 +88,9 @@ by using the *hello-rules* quickstart and this client to interact with Active MQ
-   ```
- - configure the Active MQ self-signed certificates, the steps can be found [here](https://access.redhat.com/documentation/en-us/red_hat_amq/2020.q4/html/deploying_amq_broker_on_openshift/deploying_broker-on-ocp-using-templates_broker-ocp#connecting-external-clients-to-template-based-brokers_broker-ocp)
-
+ - install the KieApp using the *Enable JMS Integration* setting as described in the  Business Automation Operator documentation [here](https://access.redhat.com/documentation/en-us/red_hat_process_automation_manager/7.8/html/deploying_a_red_hat_process_automation_manager_environment_on_red_hat_openshift_container_platform_using_operators/operator-con) at step 12.
````
```suggestion
 - install the KieApp using the *Enable JMS Integration* setting as described in the  Business Automation Operator documentation [here](https://access.redhat.com/documentation/en-us/red_hat_process_automation_manager/7.13/html/deploying_a_red_hat_process_automation_manager_environment_on_red_hat_openshift_container_platform_using_operators/operator-con) at step 12.
```

spolti commented on quickstarts/hello-rules/README.md:
````diff
@@ -125,35 +88,9 @@ by using the *hello-rules* quickstart and this client to interact with Active MQ
--p IMAGE_STREAM_NAMESPACE=openshift
-```
-
+After properly configuring the Active-MQ pre requisites, you have to wait that your KieApp will be ready and running before continuing.
````
```suggestion
After properly configuring the Active-MQ pre requisites, you have to wait the KieApp gets ready before continuing.
```

spolti commented on quickstarts/library-process/README.md:
```diff
@@ -6,79 +6,41 @@ This quickstart is intend to be used with the [IBM BAMOE Kie Server](https://git
 please take a look [here](https://access.redhat.com/RegistryAuthentication)
 
+To create the IBM BAMOE environment on your OpenShift instance you need to install Business Automation Operator.
+For instructions about installing Business Automation Operator, see [the product documentation](https://access.redhat.com/documentation/en-us/red_hat_process_automation_manager/7.8/html/deploying_a_red_hat_process_automation_manager_environment_on_red_hat_openshift_container_platform_using_operators/operator-con#operator-subscribe-proc) step.
```
```suggestion
For instructions about installing Business Automation Operator, see [the product documentation](https://access.redhat.com/documentation/en-us/red_hat_process_automation_manager/7.13/html/deploying_a_red_hat_process_automation_manager_environment_on_red_hat_openshift_container_platform_using_operators/operator-con#operator-subscribe-proc) step.
```

spolti commented on quickstarts/library-process/README.md:
```diff
@@ -6,79 +6,41 @@ This quickstart is intend to be used with the [IBM BAMOE Kie Server](https://git
+For instructions about installing Business Automation Operator, see [the product documentation](https://access.redhat.com/documentation/en-us/red_hat_process_automation_manager/7.8/html/deploying_a_red_hat_process_automation_manager_environment_on_red_hat_openshift_container_platform_using_operators/operator-con#operator-subscribe-proc) step.
 
-Deploy the `credentials secret` provided as example:
+Then access the wizard installer, instructions available [here](https://access.redhat.com/documentation/en-us/red_hat_process_automation_manager/7.8/html/deploying_a_red_hat_process_automation_manager_environment_on_red_hat_openshift_container_platform_using_operators/operator-con#operator-deploy-start-proc)
```
```suggestion
Then access the wizard installer, instructions available [here](https://access.redhat.com/documentation/en-us/red_hat_process_automation_manager/7.13/html/deploying_a_red_hat_process_automation_manager_environment_on_red_hat_openshift_container_platform_using_operators/operator-con#operator-deploy-start-proc)
```

spolti commented on quickstarts/library-process/README.md:
```diff
@@ -129,43 +91,18 @@ by using the *library-process* quickstart and this client to interact with Activ
-
-After properly configuring the Active-MQ pre-requisites, deploy the Kie Server using the AMQ Application template:
-To deploy the template execute the following command:
+- install the KieApp using the *Enable JMS Integration* setting as described in the  Business Automation Operator documentation [here](https://access.redhat.com/documentation/en-us/red_hat_process_automation_manager/7.8/html/deploying_a_red_hat_process_automation_manager_environment_on_red_hat_openshift_container_platform_using_operators/operator-con) at step 12.
```
```suggestion
- install the KieApp using the *Enable JMS Integration* setting as described in the  Business Automation Operator documentation [here](https://access.redhat.com/documentation/en-us/red_hat_process_automation_manager/7.13/html/deploying_a_red_hat_process_automation_manager_environment_on_red_hat_openshift_container_platform_using_operators/operator-con) at step 12.
```

spolti commented on quickstarts/library-process/README.md:
````diff
@@ -129,43 +91,18 @@ by using the *library-process* quickstart and this client to interact with Activ
--p IMAGE_STREAM_NAMESPACE=openshift
-```
 
+After properly configuring the Active-MQ pre-requisites, you have to wait that your KieApp will be ready and running before continuing.
````
```suggestion
After properly configuring the Active-MQ pre-requisites, you have to wait the KieApp gets ready before continuing.
```

spolti commented on templates/contrib/jdbc/README.md:
```diff
@@ -174,43 +153,14 @@ After you build your extension image you can:
-
-To create a new app using a extension image, you can go through the OpenShift web console and fill all the needed fields or
-create it using command line:
+All the steps needed are documented [here](https://access.redhat.com/documentation/en-us/red_hat_process_automation_manager/7.10/html/deploying_red_hat_process_automation_manager_on_red_hat_openshift_container_platform/operator-con_openshift-operator) 
```
```suggestion
All the steps needed are documented [here](https://access.redhat.com/documentation/en-us/red_hat_process_automation_manager/7.13/html/deploying_red_hat_process_automation_manager_on_red_hat_openshift_container_platform/operator-con_openshift-operator) 
```

spolti reviewed (changes requested):
Nice work Davide.
made some suggestions, and plus, make sure to remove the adocs generation command from the Makefile as well.

davidesalerno commented:
@spolti Thank you very much for your precious review.

I updated the links to the documentation referring to the existing ones of the 7.12 release and so using:
- https://access.redhat.com/documentation/en-us/red_hat_process_automation_manager/7.12/html/deploying_red_hat_process_automation_manager_on_red_hat_openshift_container_platform/operator-con_openshift-operator#operator-subscribe-proc_openshift-operator
- https://access.redhat.com/documentation/en-us/red_hat_process_automation_manager/7.12/html/deploying_red_hat_process_automation_manager_on_red_hat_openshift_container_platform/operator-con_openshift-operator#operator-environment-deploy-assy_openshift-operator
- https://access.redhat.com/documentation/en-us/red_hat_process_automation_manager/7.12/html/deploying_red_hat_process_automation_manager_on_red_hat_openshift_container_platform/operator-con_openshift-operator#operator-deploy-kieserver-proc_openshift-operator

and so in the code the links are referencing the 7.13 release.

Removed also the generate adocs target into the Makefile.

spolti commented:
> @spolti Thank you very much for your precious review.
> 
> I updated the links to the documentation referring to the existing ones of the 7.12 release and so using:
> 
> * https://access.redhat.com/documentation/en-us/red_hat_process_automation_manager/7.12/html/deploying_red_hat_process_automation_manager_on_red_hat_openshift_container_platform/operator-con_openshift-operator#operator-subscribe-proc_openshift-operator
> * https://access.redhat.com/documentation/en-us/red_hat_process_automation_manager/7.12/html/deploying_red_hat_process_automation_manager_on_red_hat_openshift_container_platform/operator-con_openshift-operator#operator-environment-deploy-assy_openshift-operator
> * https://access.redhat.com/documentation/en-us/red_hat_process_automation_manager/7.12/html/deploying_red_hat_process_automation_manager_on_red_hat_openshift_container_platform/operator-con_openshift-operator#operator-deploy-kieserver-proc_openshift-operator
> 
> and so in the code the links are referencing the 7.13 release.
> 
> Removed also the generate adocs target into the Makefile.

My Pleasure man!

it is fine to keep as 7.12.
